src/common/system/system_windows.cpp

```diff
@@ -119,7 +119,7 @@ std::string CSystemUtilsWindows::GetSaveDir()
     if (envUSERPROFILE == nullptr)
     {
         GetLogger()->Warn("Unable to find directory for saves - using current directory");
-        savegameDir = CSystemUtils::GetSaveDir(false);
+        savegameDir = CSystemUtils::GetSaveDir();
```

Comment:
Whoops, my bad, I changed my mind about this code so many times that I left this here accidentaly :P This should be `"./saves"` to keep it the same as in other system_* files